puppet6-bootstrap.ps1

```diff
@@ -86,4 +87,9 @@ Write-Host "Puppet successfully installed."
 if($env:Path -notcontains 'C:\Program Files\Puppet Labs\Puppet\bin' ) {
   $env:Path += ';C:\Program Files\Puppet Labs\Puppet\bin'
   [Environment]::SetEnvironmentVariable('Path', $env:Path, 'Machine')
+}
+
+if ($PuppetServerPort) {
+    $set_port_args = @('config', 'set', 'masterport', $PuppetServerPort, '--section', 'main')
+    Start-Process -FilePath puppet.bat -ArgumentList $set_port_args -Wait -PassThru
```

Comment:
You could probably do away with the start-process 😄  (just have `puppet config set masterport $PuppetServerPort --section agent` should work.) 

_I suspect we use Start-Process in that script only because msiexec doesn't block or something...._

Reply:
@nicolasvan better in 865935a?

Reply:
yup I'd go with 865935a 👍